projects/route-analyzer/src/lib/route-analyzer.ts

```diff
@@ -301,4 +307,20 @@ function fixupLoadChildren(
         }
         return referencedModule.module;
     }
+
+    // Using absolute path prevents issues when two module files have the exact same name
+    function getLoadChildrenAbsolutePath(moduleFolderPath: string, loadChildrenImportPath: string): string {
+        let absolutePath: string;
```

Comment:
Nitpicking since `const` is preferred

```ts
const absolutePath = path.isAbsolute(moduleFolderPath 
   ?  path.join(moduleFolderPath, loadChildrenImportPath) 
   :  path.join(programDirectory, moduleFolderPath, loadChildrenImportPath);

// Must append the file extension, as the TS imports never include it
return `${absolutePath}.ts`;
```